# Real Time information Sources for Tropical Cyclones

Real time information for Tropical cyclones are available from various sources. These depend on the corresponding basin since there are different designated service providers for each basin according to the World Meteological Organization (WMO). These are called 'Regional Specialized Meteorological Centres' (RSMC). In addition to those, there are Tropical Cyclone Warning Centers (TCWCs) that also provide information sometimes different from the RSMC's but generally cover a broader region. Naturally there are also the national services called 'National Meteorological and Hydrological Services' (NMHSs) which might be limited in scope. 

One can find more info regarding these services in http://severe.worldweather.wmo.int/.

The sources we rely on for now are from the National Hurricane Center (NHC), part of National Weather Service (NWS) in the US administrated by the National Oceanic & Atmospheric Administration (NOAA). More info at http://www.noaa.gov/.

The NHC is located at http://www.nhc.noaa.gov/ for Atlantic (including Caribbean Sea-Gulf of Mexico) and Easter Pacific Basins. Note that according to WMO this is the designated RSMC based in Miami, FL USA.

For Central North Pacific the corresponding RMSC is again NWS but now based in Honolulu, Hawaii http://www.prh.noaa.gov/pr/hnl/cphc/.

For western north pacific the designated RSMC is the Tokyo-Typhoon Center (http://www.jma.go.jp/en/typh/index.html) but the Joint Typhoon Warning Center (JTWC) based also in Hawaii is providing bulletins (Warnings) for the general Pacific and Indian basins. Note that the JTWC is part of the US military service made availbale to the public http://www.usno.navy.mil/. 

For the north Indian Ocean, the RMSC is the Indian Meteorological Dept. with info and data found at http://www.rsmcnewdelhi.imd.gov.in/index.php?lang=en.

Finally for the area around Australia the TCWC provider is the Australian Bureau of Meteorology http://www.bom.gov.au/cyclone/index.shtml.

Note that the above sources are deemed 'operational' and provide official information. An aggregate service for all available daily bulletins can be found at http://weather.unisys.com/hurricane/ (see archive link therein). 

## North Atlantic - Caribbean - Eastern Pacific.

The information in the form of bulletins comes from NOAA. At the corresponding website  http://www.nhc.noaa.gov/ information are available in a variety of forms. We download the bulletins from the corresponding RSS feeds. Namely for the referenced basins the feeds are

 Atlantic Basin Tropical Cyclones: http://www.nhc.noaa.gov/index-at.xml
 Eastern Pacific Basin Tropical Cyclones: http://www.nhc.noaa.gov/index-ep.xml
 
 More info can be found in http://www.nhc.noaa.gov/aboutrss.shtml.
 
 The data are stored locally at '\\139.191.244.103\g$\inetpub\wwwroot\gdacsold\cyclones\NOAA'. There each Tropical Cyclone (TC) is given a  designation based on the basin ('Eastern Pacific' = 'EP', 'Central Pacific' = 'CP', 'Atlantic' = 'AL') and a number (increasing) refering to sequence of appearance in the specific year , eg. '032015' means the 3rd one in that basin in 2015.
 
We can view the corresponding bulletins (if available) like this  

In [1]:
import feedparser

url='http://www.nhc.noaa.gov/index-at.xml'

atl=feedparser.parse(url)

In [2]:
data=atl['entries']
data[0]['summary']

u'<br />\n000<br />\nABNT20 KNHC 300535<br />\nTWOAT <br />\n<br />\nTROPICAL WEATHER OUTLOOK<br />\nNWS NATIONAL HURRICANE CENTER MIAMI FL<br />\n200 AM EDT TUE AUG 30 2016<br />\n<br />\nFor the North Atlantic...Caribbean Sea and the Gulf of Mexico:<br />\n<br />\nThe National Hurricane Center is issuing advisories on Hurricane<br />\nGaston, located well east of Bermuda, on Tropical Depression Eight,<br />\nlocated southeast of Cape Hatteras, North Carolina, and on Tropical<br />\nDepression Nine, located over the southeastern Gulf of Mexico.<br />\n<br />\nA weak area of low pressure, associated with a tropical wave, is<br />\nlocated over the far eastern Atlantic between the west coast of<br />\nAfrica and the Cabo Verde Islands.  Environmental conditions are<br />\nexpected to become a little more favorable for some gradual<br />\ndevelopment of this system late this week while it moves<br />\nwestward at 15 to 20 mph over the tropical Atlantic.<br />\n* Formation chance through 

In [3]:
import feedparser

url='http://www.nhc.noaa.gov/index-ep.xml'

ep=feedparser.parse(url)

In [4]:
data=ep['entries']
data[0]['summary']

u'<br />\n000<br />\nABPZ20 KNHC 300537<br />\nTWOEP <br />\n<br />\nTROPICAL WEATHER OUTLOOK<br />\nNWS NATIONAL HURRICANE CENTER MIAMI FL<br />\n1100 PM PDT MON AUG 29 2016<br />\n<br />\nFor the eastern North Pacific...east of 140 degrees west longitude:<br />\n<br />\nThe National Hurricane Center is issuing advisories on Hurricane<br />\nLester, located well east of the Hawaiian Islands.<br />\n<br />\nA broad and elongated area of low pressure located several hundred<br />\nmiles southwest of the southern tip of the Baja California peninsula<br />\ncontinues to produce a large area of cloudiness and disorganized<br />\nshowers.  Development of this system is not expected as it moves<br />\nlittle during the next few days.<br />\n* Formation chance through 48 hours...low...near 0 percent<br />\n* Formation chance through 5 days...low...near 0 percent<br />\n<br />\nAn area of low pressure could form a few hundred miles south or<br />\nsouthwest of the south-central coast of Mexico

## Central Pacific

Again NOAA provides info from the dedicated site. We collect the data from the RSS feed of the corresponding website http://www.prh.noaa.gov/hnl/cphc/ that is http://www.prh.noaa.gov/cphc/index-cp.xml.

In [5]:
import feedparser

url='http://www.prh.noaa.gov/cphc/index-cp.xml'

cp=feedparser.parse(url)

In [6]:
data=cp['entries']
data[0]['summary']


u'Issued at "0300 UTC TUE AUG 30 2016"'

## West Pacific

From the JTWC web site the bulletins can be accessed in various formats. We download the bulletin in JMV 3.0 Data format. These bulletins are stored locally at the following location '\\139.191.244.103\g$\inetpub\wwwroot\gdacsold\cyclones\JTCW'. There  they are referenced through the usual naming system (see above). A typical sample is

WTPN51 PGTW 040900    
WARNING    ATCG MIL 13W NWP 150804072931
2015080406 13W SOUDELOR   021  02 290 13 SATL 010
T000 186N 1383E 140 R064 040 NE QD 035 SE QD 035 SW QD 035 NW QD R050 095 NE QD 070 SE QD 070 SW QD 095 NW QD R034 185 NE QD 155 SE QD 150 SW QD 185 NW QD 
T012 193N 1363E 140 R064 040 NE QD 035 SE QD 035 SW QD 035 NW QD R050 085 NE QD 070 SE QD 070 SW QD 085 NW QD R034 165 NE QD 145 SE QD 140 SW QD 165 NW QD 
T024 199N 1343E 140 R064 040 NE QD 040 SE QD 040 SW QD 040 NW QD R050 080 NE QD 070 SE QD 070 SW QD 080 NW QD R034 155 NE QD 140 SE QD 135 SW QD 155 NW QD 
T036 206N 1322E 135 R064 040 NE QD 040 SE QD 040 SW QD 040 NW QD R050 080 NE QD 070 SE QD 070 SW QD 080 NW QD R034 150 NE QD 135 SE QD 135 SW QD 150 NW QD 
T048 214N 1298E 130 R064 045 NE QD 040 SE QD 040 SW QD 040 NW QD R050 080 NE QD 070 SE QD 070 SW QD 080 NW QD R034 150 NE QD 135 SE QD 135 SW QD 150 NW QD 
T072 231N 1251E 120 R064 045 NE QD 040 SE QD 040 SW QD 045 NW QD R050 075 NE QD 070 SE QD 070 SW QD 075 NW QD R034 145 NE QD 135 SE QD 135 SW QD 145 NW QD 
T096 253N 1207E 100 R034 015 NE QD 015 SE QD 000 SW QD 015 NW QD 
T120 279N 1177E 065 R034 010 NE QD 010 SE QD 000 SW QD 010 NW QD 
AMP
SUBJ:  SUPER TYPHOON 13W (SOUDELOR) WARNING NR 021    
1. SUPER TYPHOON 13W (SOUDELOR) WARNING NR 021    
   02 ACTIVE TROPICAL CYCLONES IN NORTHWESTPAC
   MAX SUSTAINED WINDS BASED ON ONE-MINUTE AVERAGE
   WIND RADII VALID OVER OPEN WATER ONLY
    ---
   WARNING POSITION:
   040600Z --- NEAR 18.6N 138.3E
     MOVEMENT PAST SIX HOURS - 290 DEGREES AT 13 KTS
     POSITION ACCURATE TO WITHIN 010 NM
     POSITION BASED ON EYE FIXED BY SATELLITE
   PRESENT WIND DISTRIBUTION:
   MAX SUSTAINED WINDS - 140 KT, GUSTS 170 KT
   WIND RADII VALID OVER OPEN WATER ONLY
   RADIUS OF 064 KT WINDS - 040 NM NORTHEAST QUADRANT
                            035 NM SOUTHEAST QUADRANT
                            035 NM SOUTHWEST QUADRANT
                            035 NM NORTHWEST QUADRANT
   RADIUS OF 050 KT WINDS - 095 NM NORTHEAST QUADRANT
                            070 NM SOUTHEAST QUADRANT
                            070 NM SOUTHWEST QUADRANT
                            095 NM NORTHWEST QUADRANT
   RADIUS OF 034 KT WINDS - 185 NM NORTHEAST QUADRANT
                            155 NM SOUTHEAST QUADRANT
                            150 NM SOUTHWEST QUADRANT
                            185 NM NORTHWEST QUADRANT
   REPEAT POSIT: 18.6N 138.3E
    ---
   FORECASTS:
   12 HRS, VALID AT:
   041800Z --- 19.3N 136.3E
   MAX SUSTAINED WINDS - 140 KT, GUSTS 170 KT
   WIND RADII VALID OVER OPEN WATER ONLY
   RADIUS OF 064 KT WINDS - 040 NM NORTHEAST QUADRANT
                            035 NM SOUTHEAST QUADRANT
                            035 NM SOUTHWEST QUADRANT
                            035 NM NORTHWEST QUADRANT
   RADIUS OF 050 KT WINDS - 085 NM NORTHEAST QUADRANT
                            070 NM SOUTHEAST QUADRANT
                            070 NM SOUTHWEST QUADRANT
                            085 NM NORTHWEST QUADRANT
   RADIUS OF 034 KT WINDS - 165 NM NORTHEAST QUADRANT
                            145 NM SOUTHEAST QUADRANT
                            140 NM SOUTHWEST QUADRANT
                            165 NM NORTHWEST QUADRANT
   VECTOR TO 24 HR POSIT: 290 DEG/ 10 KTS
    ---
   24 HRS, VALID AT:
   050600Z --- 19.9N 134.3E
   MAX SUSTAINED WINDS - 140 KT, GUSTS 170 KT
   WIND RADII VALID OVER OPEN WATER ONLY
   RADIUS OF 064 KT WINDS - 040 NM NORTHEAST QUADRANT
                            040 NM SOUTHEAST QUADRANT
                            040 NM SOUTHWEST QUADRANT
                            040 NM NORTHWEST QUADRANT
   RADIUS OF 050 KT WINDS - 080 NM NORTHEAST QUADRANT
                            070 NM SOUTHEAST QUADRANT
                            070 NM SOUTHWEST QUADRANT
                            080 NM NORTHWEST QUADRANT
   RADIUS OF 034 KT WINDS - 155 NM NORTHEAST QUADRANT
                            140 NM SOUTHEAST QUADRANT
                            135 NM SOUTHWEST QUADRANT
                            155 NM NORTHWEST QUADRANT
   VECTOR TO 36 HR POSIT: 290 DEG/ 10 KTS
    ---
   36 HRS, VALID AT:
   051800Z --- 20.6N 132.2E
   MAX SUSTAINED WINDS - 135 KT, GUSTS 165 KT
   WIND RADII VALID OVER OPEN WATER ONLY
   RADIUS OF 064 KT WINDS - 040 NM NORTHEAST QUADRANT
                            040 NM SOUTHEAST QUADRANT
                            040 NM SOUTHWEST QUADRANT
                            040 NM NORTHWEST QUADRANT
   RADIUS OF 050 KT WINDS - 080 NM NORTHEAST QUADRANT
                            070 NM SOUTHEAST QUADRANT
                            070 NM SOUTHWEST QUADRANT
                            080 NM NORTHWEST QUADRANT
   RADIUS OF 034 KT WINDS - 150 NM NORTHEAST QUADRANT
                            135 NM SOUTHEAST QUADRANT
                            135 NM SOUTHWEST QUADRANT
                            150 NM NORTHWEST QUADRANT
   VECTOR TO 48 HR POSIT: 290 DEG/ 12 KTS
    ---
   EXTENDED OUTLOOK:
   48 HRS, VALID AT:
   060600Z --- 21.4N 129.8E
   MAX SUSTAINED WINDS - 130 KT, GUSTS 160 KT
   WIND RADII VALID OVER OPEN WATER ONLY
   RADIUS OF 064 KT WINDS - 045 NM NORTHEAST QUADRANT
                            040 NM SOUTHEAST QUADRANT
                            040 NM SOUTHWEST QUADRANT
                            040 NM NORTHWEST QUADRANT
   RADIUS OF 050 KT WINDS - 080 NM NORTHEAST QUADRANT
                            070 NM SOUTHEAST QUADRANT
                            070 NM SOUTHWEST QUADRANT
                            080 NM NORTHWEST QUADRANT
   RADIUS OF 034 KT WINDS - 150 NM NORTHEAST QUADRANT
                            135 NM SOUTHEAST QUADRANT
                            135 NM SOUTHWEST QUADRANT
                            150 NM NORTHWEST QUADRANT
   VECTOR TO 72 HR POSIT: 290 DEG/ 12 KTS
    ---
   72 HRS, VALID AT:
   070600Z --- 23.1N 125.1E
   MAX SUSTAINED WINDS - 120 KT, GUSTS 145 KT
   WIND RADII VALID OVER OPEN WATER ONLY
   RADIUS OF 064 KT WINDS - 045 NM NORTHEAST QUADRANT
                            040 NM SOUTHEAST QUADRANT
                            040 NM SOUTHWEST QUADRANT
                            045 NM NORTHWEST QUADRANT
   RADIUS OF 050 KT WINDS - 075 NM NORTHEAST QUADRANT
                            070 NM SOUTHEAST QUADRANT
                            070 NM SOUTHWEST QUADRANT
                            075 NM NORTHWEST QUADRANT
   RADIUS OF 034 KT WINDS - 145 NM NORTHEAST QUADRANT
                            135 NM SOUTHEAST QUADRANT
                            135 NM SOUTHWEST QUADRANT
                            145 NM NORTHWEST QUADRANT
   VECTOR TO 96 HR POSIT: 300 DEG/ 11 KTS
    ---
   LONG RANGE OUTLOOK:
    ---
   96 HRS, VALID AT:
   080600Z --- 25.3N 120.7E
   MAX SUSTAINED WINDS - 100 KT, GUSTS 125 KT
   WIND RADII VALID OVER OPEN WATER ONLY
   VECTOR TO 120 HR POSIT: 315 DEG/ 09 KTS
    ---
   120 HRS, VALID AT:
   090600Z --- 27.9N 117.7E
   MAX SUSTAINED WINDS - 065 KT, GUSTS 080 KT
   WIND RADII VALID OVER OPEN WATER ONLY
    ---
REMARKS:
040900Z POSITION NEAR 18.8N 137.8E.
SUPER TYPHOON 13W (SOUDELOR), LOCATED APPROXIMATELY 750 NM
SOUTHEAST OF KADENA AB, HAS TRACKED WEST-NORTHWESTWARD AT 13
KNOTS OVER THE PAST SIX HOURS.
MAXIMUM SIGNIFICANT WAVE HEIGHT AT 040600Z IS 48 FEET. NEXT
WARNINGS AT 041500Z, 042100Z, 050300Z AND 050900Z.
REFER TO TROPICAL DEPRESSION 14W (FOURTEEN) WARNINGS (WTPN32 PGTW)
FOR SIX-HOURLY UPDATES.
//
1315072718 134N1750E  15
1315072800 133N1732E  15
1315072806 133N1716E  15
1315072812 133N1698E  15
1315072818 128N1679E  15
1315072900 124N1658E  15
1315072906 124N1639E  20
1315072912 123N1627E  20
1315072918 124N1617E  20
1315073000 127N1610E  25
1315073006 136N1599E  30
1315073012 137N1591E  35
1315073018 137N1586E  40
1315073100 136N1576E  40
1315073106 136N1566E  40
1315073112 136N1556E  35
1315073118 136N1543E  35
1315080100 138N1530E  35
1315080106 140N1521E  40
1315080112 141N1508E  45
1315080118 142N1498E  50
1315080200 146N1483E  60
1315080200 146N1483E  60
1315080206 148N1468E  75
1315080206 148N1468E  75
1315080206 148N1468E  75
1315080212 151N1459E  90
1315080212 151N1459E  90
1315080212 151N1459E  90
1315080218 156N1450E 105
1315080218 156N1450E 105
1315080218 156N1450E 105
1315080300 162N1441E 115
1315080300 162N1441E 115
1315080300 162N1441E 115
1315080306 169N1429E 115
1315080306 169N1429E 115
1315080306 169N1429E 115
1315080312 175N1418E 135
1315080312 175N1418E 135
1315080312 175N1418E 135
1315080318 178N1407E 155
1315080318 178N1407E 155
1315080318 178N1407E 155
1315080400 182N1396E 155
1315080400 182N1396E 155
1315080400 182N1396E 155
1315080406 186N1383E 140
1315080406 186N1383E 140
1315080406 186N1383E 140
NNNN

JTWC also has a post-analysis for Best Track information at http://www.usno.navy.mil/NOOC/nmfc-ph/RSS/jtwc/best_tracks/ which however are updated annualy (For more info see [Best Track](BestTrack.ipynb)). 

The US military has an extended research division that provides also additional resources including satelite derived data. These are available at the Naval research Laboratory (http://www.nrlmry.navy.mil/). These services include the Automated Tropical Cyclone Forecasting System (ATCF - http://www.nrlmry.navy.mil/atcf_web/index1.html ) which provides an computer automated process for operational users such as the JTWC, NHC, HWRF etc. They also have Numerical  Weather Forecasting for global weather and TC in particular ( http://www.nrlmry.navy.mil/coamps-web/web/tc ). For more info see [NRL](NRL.ipynb).

The main source is the JTWC. However the Tokyo Typhoon Center (TTC) issues also bulletins. These can be found at their own website http://www.jma.go.jp/en/typh/ where they can be also be retrieved from an 'experimental' RSS feed at http://www.data.jma.go.jp/fcd/yoho/cap-rsmctk/atom.xml. These info are also available at the WMO since the TTC is the official RMSC for the westerm pacific. 

We can read the feed from the above link as (the current info will be displayed if available)

In [7]:
import feedparser

url='http://www.data.jma.go.jp/fcd/yoho/cap-rsmctk/atom.xml'

jmafeed=feedparser.parse(url)


data=jmafeed['entries']
data[0]['summary']

u'<pre>TYPHOON LIONROCK (1610)\nANALYSIS (3006UTC):\nPressure 965 hPa\nat 37.7 degree north 142.0 degree east\nposition FAIR (radius of 55 to 110 km).\nMoving north-north-west 21 knots.\nMax winds 65 knots.\nGust 95 knots.\n30 knot wind radius 600 km south 440 km north.\n50 knot wind radius 220 km north-east 110 km south-west.\n</pre>'

In addition TTC provides re-analysis data for past events in the form of 'Best Track Data' at  http://www.jma.go.jp/jma/jma-eng/jma-center/rsmc-hp-pub-eg/trackarchives.html. (For more info see [Best Track](BestTrack.ipynb)) Furthermore the newly operational Himawari satellite provides satellite images for the globe and more specific areas. These are available online at http://www.jma.go.jp/en/gms/index.html?area=6&element=0&mode=UTC. For more info see http://ds.data.jma.go.jp/mscweb/data/himawari/index.html

## Additional services

In addition to the operational services above there are websites that gather available data and provide relative information. Such sites are

The Pacific Disaster Ceneter http://www.pdc.org/ : Information mainly from JTWC

Weather Underground  http://www.wunderground.com/  : Incorporates NOAA data see http://www.wunderground.com/wundermap/, part of Weather Channel.

NOAA environment visualization laboratory http://www.nnvl.noaa.gov/Default.php : for animations and satelite images


## Numerical Weather Forecasts (NWF)

Another recent source of information comes from global numerical simulations. These are performed by various organizations and some are deemed 'operational' while others are 'experimental'. The two main organization that produce global analysis and forecasting are the European Centre for Medium-Range Weather Forecasts (ECMWF) and NOAA (various divisions). 

ECMWF (http://www.ecmwf.int/) provides a plethora of simulated data including short and long-term forecasting. However unlike NOAA (see below) their data are not always available for free. For more info on available TC ECMWF data look at [ECMWF data](ECMWF data.ipynb). 

NOAA has several divisions that utilize a variety of models to compute various quantities. The more pertinent one is the Hurricane Weather Research and Forecast (HWRF). From HWRF website (http://www.emc.ncep.noaa.gov/gc_wmb/vxt/HWRF/) we read


"The HWRF system is composed of the WRF model software infrastructure, the Non-Hydrostatic Mesoscale Model (NMM) dynamic core, the three-dimensional Princeton Ocean Model (POM), the NCEP coupler, and a physics suite tailored to the tropics, including air-sea interactions over warm water and under high wind conditions, and boundary layer and cloud physics developed for hurricane forecasts. The atmosphere-ocean coupled HWRF system consists of multiple movable two-way interactive nested grids that follow the projected path of the storm. Atmospheric component of the HWRF model was coupled to the Princeton Ocean Model (POM) developed by GFDL/URI using a sophisticated coupler developed at NCEP for providing accurate representation of air-sea interactions. An advanced vortex initialization scheme and NCEP GSI based HWRF Data Assimilation System (HDAS) provide means to represent the initial location, intensity, size and structure of the inner core of a hurricane and it’s large-scale environment. The NCEP Global Forecast System (GFS) analysis and forecasts provide initial and boundary conditions for the HWRF model. POM uses a feature based initialization procedure for representing oceanic features such as the loop current, warm/cold core rings and the cold wake generated by the storm."

HWRF are providing operational info for TC in all basins since 2015. They also publish their raw data (every 6 hours) for the last 2 timestamps at http://ftpprd.ncep.noaa.gov/data/nccf/com/hur/prod/ and ftp://ftpprd.ncep.noaa.gov/pub/data/nccf/com/hur/prod/. Note that in the above links the GFS data mentioned above are also given. However, unlike HWRF, the GFS data are available for longer periods of time in various outlets. For more info see http://nomads.ncep.noaa.gov/ 

In addition the HWRF model is open sourced in the following location http://www.dtcenter.org/HurrWRF/users/. 

"The WRF system contains two dynamical solvers, referred to as the ARW (Advanced Research WRF) core and the NMM (Nonhydrostatic Mesoscale Model) core. The ARW has been largely developed and maintained by the Mesoscale and Microscale Meteorology (MMM) Laboratory of the National Center for Atmospheric Research (NCAR), and its users' page is: ( http://www2.mmm.ucar.edu/wrf/users/ ). The NMM core was developed by the National Centers for Environmental Prediction, and its user support is provided by the Developmental Testbed Center. Its users' page is: http://www.dtcenter.org/wrf-nmm/users/WRF-NMM.

NCAR is actually running a Tropical Cyclone Guidance Project with a lot of info and links available (http://www.ral.ucar.edu/hurricanes/). In particular info regarding another product provided in the operation HWRF page (see above) called Statistical Hurricane Intensity Prediction Scheme (SHIPS) can be found here http://rammb.cira.colostate.edu/research/tropical_cyclones/ships/ and here http://www.ral.ucar.edu/hurricanes/realtime/ .

Another numerical Hurricane model that also provides modules for HWRF is the Geophysical Fluid Dynamics Laboratory's (GFDL) model. Info and real time forecast can be found here http://data1.gfdl.noaa.gov/hurricane/gfdl_ensemble/GFE_DEMO_2015.php?STORMID=09W&YMDH=2015070800_CHAN-HOM09W&prod=H1 and http://www.gfdl.noaa.gov/hurricane-portal.

JMA : http://www.jma.go.jp/jma/jma-eng/jma-center/nwp/nwp-top.htm

## Satellite Driven Analysis

Another NOAA division that produces an composite product based on satellite information is the NOAA Satellite and Information Service (NESDIS - http://www.nesdis.noaa.gov/). Their product related to TCs is called 'Multiplatform Tropical Cyclone Surface Winds Analysis (MTCSWA)' that provides satellite based wind estimates around TCs. According to their site 

"This product combines information from five data sources to create a mid-level (near 700 hPa) wind analysis using a variational approach described in Knaff and DeMaria (2006). The resulting mid-level winds are then adjusted to the surface applying a very simple single column approach. Over the ocean an adjustment factor is applied, which is a function of radius from the center ranging from 0.9 to 0.7, and the winds are turned 20 degrees toward low pressure. Over land, the oceanic winds are reduced by an additional 20% and turned an additional 20 degrees toward low pressure.

The six datasets used are the QuikSCAT scatterometer, which is adjusted upward to 700 hPa in the same manner as the surface winds are adjusted downward, ASCAT scatterometer (treated the same way as QuikSCAT), feature track winds in the mid-levels from the operational satellite centers, 2-d flight-level winds estimated from infrared imagery (see Mueller et al 2006 ) and 2-d winds created from Advanced Microwave Sounding Unit (AMSU)- derived height fields and solving the non-linear balance equations as described in Bessho et al (2006).

Each of the input data are show in subpannels following the analysis (i.e., storm-relative). Shown are AMSU winds, Cloud-drift/IR/WV winds, IR-proxy winds and Scatterometer winds; QuikSCAT (BLUE) and ASCAT (RED). All input data in these pannels has been reduced to a 10-m land or oceanic exposurenon depending on the analysis time location (i.e., non-surface data has been reduced to a 10-m exposure)."

An extensive set of data can be found in the form of pictures at their development website in Colorado State University (http://rammb.cira.colostate.edu/products/tc_realtime/), while the raw data are available per TC at this address 
ftp://satepsanone.nesdis.noaa.gov/MTCSWA/. Note that availability of data has an expiration date (roughly a month or so).

Information regarding additional TC products can be found at http://www.ssd.noaa.gov/PS/TROP/index.html. These include except the MTCSWA above the so called Advanced Dvorak Technique (ADT). For more see the Development site at the University of Winsconsin-Madison (http://tropic.ssec.wisc.edu/misc/adt/info.html). The description of the method there states

"The Advanced Dvorak Technique (ADT) utilizes longwave-infrared, temperature measurements from geostationary satellites to estimate tropical cyclone (TC) intensity. The ADT currently utilizes an objective storm center determination scheme and cloud pattern determination logic to remove the subjectivity aspect from the intensity estimation process. It also can be applied to all phases of the TC lifecycle; something that previous objective schemes could not do. Finally, the ADT makes use of statistical analysis results obtained from a 10+ year sample of North Atlantic storms, along with a significant sample of West and East Pacific storms, covering the entire spectrum of TC intensities to derive a regression-based intensity value estimate for various phases of the TC lifecycle."

The main page for TCs of the University of Winsconsin-Madison also serves as a portal to information for active TCs http://tropic.ssec.wisc.edu/. A click on a current TC launches a viewer where all current satellite driven information can be overlayed. 

# Satellite data

The most important info needed for analysis of a TC is the pressure and wind field around it. In terms of wind field the satellite analysis uses a number of satellites and technics for computing the surface wind vectors (SW). All the available sources with graphs can be found at http://manati.star.nesdis.noaa.gov/datasets/ASCATData.php and include the currently operational ASCAT (METOP-A,B) RapidSCAT (http://www.nasa.gov/mission_pages/station/research/experiments/1067.html ) and WindSAT. One can click on the pixeled graph for a close-up. In addition by clicking on the corresponding link of the service and navigate to 'storm data' one can get graphs for specific TC in all basins ( e.g. http://manati.star.nesdis.noaa.gov/datasets/WindSATData.php?parname=storm ). The actual data can be found in the OPeNDAP service run by NASA at this address  http://podaac.jpl.nasa.gov/datasetlist where one can choose the prefered 'Collection', e.g. 'Quick Scatterometer' would lead us to  http://podaac.jpl.nasa.gov/datasetlist?ids=Collections&values=QUIKSCAT&view=list where we can choose the QuickSCAT data server. 

The same way we can get to 

rapidscat: http://podaac-opendap.jpl.nasa.gov/opendap/allData/rapidscat/L2B12/v1.1/2015/

ascat(meteop b) : http://podaac-opendap.jpl.nasa.gov/opendap/allData/ascat/preview/L2/metop_b/coastal_opt/

ascat(mteop a) : http://podaac-opendap.jpl.nasa.gov/opendap/allData/ascat/preview/L2/metop_a/coastal_opt/

windsat : http://podaac-opendap.jpl.nasa.gov/opendap/allData/windsat/L3/rss/v7/

Information about available satellite data can also be found at the eSurge website http://www.storm-surge.info/data-access. Windsat data are also given as graphs in reference to TCs here http://www.remss.com/storm-watch .

One of the newest satellites in operation is the Japanese himawari-8 that covers the Central & Western Pacific. A four weeks archive can be found at http://rammb.cira.colostate.edu/ramsdis/online/himawari-8.asp#Japan. Additional imagery by other sources can be found in http://www.ssd.noaa.gov/imagery/ .

Data and info regarding scatterometers can also be accessed via the Royal Netherlands Meteorogical Institute http://www.knmi.nl/scatterometer/.

# Simplified models

## Dvorak technique

## Holland Parameter

## Simplified Holland Parameter

# Model comparison - Analysis

# Discussions - Suggestions

# Misc

NOAA among other organizations is developing a program for aggregating all their data in ERDDAP servers. Such servers can be found here http://upwell.pfeg.noaa.gov/erddap/index.html and here http://coastwatch.pfeg.noaa.gov/erddap/index.html.  Note that this is a work in progress. A similar server is actually operational here in JRC (  https://bluehub.jrc.ec.europa.eu/erddap/index.html ) where a variety of NOAA data can be found.  

A repository of NOAA data can also be found here http://www.nco.ncep.noaa.gov/pmb/products/ .

A variety of maps and images (some related to TCs) can be found here http://earthobservatory.nasa.gov/?eocn=topnav&eoci=home and here http://www.nasa.gov/mission_pages/hurricanes/main/index.html .

A nice up to date portal at wikipedia with info, pictures and links https://en.wikipedia.org/wiki/Portal:Tropical_cyclones.

JTWC bulletins archive in NOOA http://weather.noaa.gov/pub/data/raw/ with the folders refering to the code of the TC e.g. 'WTPN31 PGTW 060900' ->  http://weather.noaa.gov/pub/data/raw/wt/wtpn31.pgtw..txt.